app/assets/javascripts/add-forms.js

```diff
@@ -0,0 +1,55 @@
+$(document).on('ready page:load', function(){
```

Comment:
If you're using turbolinks, page:load should fire in all cases. If you're not, just ready is fine. I'd probably jusr turn off turbolinks